packages/Popover/src/Popover.js

```diff
@@ -18,14 +16,11 @@ import Tip from "./components/Tip/Tip";
 
 import PopoverStyled from "./Popover.styles";
 
-// ACCESSIBILITY
-// NOTE: When closing the popover seems to be better to focus the trigger button
-//       only when the close method is the ESC key, when clicking outside doesn't feel a fluid action
-//       assigning the focus to the button again
-
 const openDelay = 350;
 const closeDelay = 150;
 const throttleDelay = 20;
+const focusableElementSelector =
+  'button, [href], input, select, textarea, [tabindex]:not([tabindex="-1"]), [role="button"]';
```

Comment:
Just `'button, [href], input, select, textarea, [tabindex]'` should work, and is "cleaner", but 🤷‍♂️

Reply:
Yeah I saw that one. It is a bit easier to read, but no point in touching elements that already have the value we want